src/main/java/org/radarcns/management/config/OAuth2ServerConfiguration.java

```diff
@@ -174,7 +181,11 @@ protected AuthorizationCodeServices authorizationCodeServices() {
 
         @Bean
         public ApprovalStore approvalStore() {
-            return new JdbcApprovalStore(dataSource);
+            if (jpaProperties.getDatabase().equals(POSTGRESQL)) {
```

Comment:
If the new ApprovalStore also works for H2, perhaps rename it to `SanitizedJdbcApprovalStore` And remove this if/else branch?

Reply:
It doesn't. That's why the if else was added later on. H2 doesn't like quotes.

Reply:
That's unfortunate... Would using hibernate solve that? Or would that be too large of a refactor?

Reply:
My guess would be not large. Currently, we do not have entities maintained by hibernate in for `oauth` related tables or in other words, we only have the database schema given by spring added. The rest are handled by `spring-security-oauth`. I assume having a local entity shouldn't effect the rest. Maybe a separate PR would be better.

Reply:
Okay, a separate PR seems fine.